Clients/GUI/wwwroot/css/site.css

```diff
@@ -246,3 +246,25 @@ app {
     .cur-icon .gbp-icon {
         background-image: url(/images/currencies/gbp.png)
     }
+
+#container {
+    overflow: hidden;
+    position: relative;
+    width: 100%;
+    margin-bottom: 20px;
+    margin-top: 20px;
+    margin-left: auto;
+    margin-right: auto;
+}
+
+.leftimage {
+    width: 50%;
+    display: inline-block;
+    float: left;
+}
+
+.righttext {
```

Comment:
Maybe use class from bootstrap .text-right

Reply:
That was in the past version, I will delete it